docs/asciidoc/modules/ROOT/pages/background-operations/triggers.adoc

```diff
@@ -451,6 +451,17 @@ CALL apoc.trigger.list
 SET c.count = c.count + size([f IN $deletedNodes WHERE id(f) > 0])" | {}       | {}     | TRUE      | FALSE
 |===
 
+
+[WARNING]
+====
+Please note that, since the trigger operations are eventually consistent (based on `apoc.trigger.refresh` configuration), 
+the `apoc.trigger.show` may return some triggers not yet added/updated/removed.
```

Comment:
"actually working triggers" vs "currently working triggers"

I don't mind which word we use in the descriptions, but if possible let's keep using the same one

Reply:
Ok, I'll use currently :)